### DISCUSSION TITLE
Support extends in overrides

### DISCUSSION THREAD

**Discussion by unional** [biomejs/biome#1907] [Ideas]
Currently, `extends` can only be defined at top level.
This does not allow rules to be reused in overrides, specifically in monorepo.

For example:

```jsonc
{
  "extends": ["./node_modules/@repobuddy/biome/recommended.json"],
  "files": { ... },
  "overrides": [
    {
      "include": ["packages/low-level/src"],
      "extends": ["./node_modules/@repobuddy/biome/performant.json"]
    }
  ]
}
```

**Comment by ematipico** [comment]
Can't you write the override in the configuration you're extending?

**Reply by unional** [reply]
Cannot do that because the override include is repository specific, while the configuration is not.

The configuration can be used in any repositories.

**Comment by unional** [comment]
I'm trying to do it a different way but it is also not working:

```json5
// biome.json
{
  "extends": [".biome/default.json", ".biome/performant.json"]
}

// .biome/default.json
{
  "extends": ["@repobuddy/biome/recommended.json"],
  "files": { /* ... the usual ... */ }
}

// .biome/performant.json
{
  "extends": ["@repobuddy/biome/performant.json"],
  "files": { "includes": ["packages/low-level/src"] }
}
```

It seems like the `files` in the extends config is not being honored.
If we can support that and it is easier to do, this can be used as a workaround.

**Comment by lightninglu10** [comment]
Did we ever get support for this, or no?

**Reply by Conaclos** [reply]
This is not planned. We are reluctant to make configuration more complex than it already is. If the community shows interest in this feature, we may re-evaluate our position.
